Documentation/ksmbd.addshare.8

```diff
@@ -0,0 +1,52 @@
+.TH KSMBD.ADDSHARE "8" "October 2021" "Usage: smbshareadd" "Linux System Administration"
```

Comment:
please change smbshareadd to ksmbd.addshare. There is a patch to change smbshareadd to ksmbd.addshare from Enzo.

Reply:
fixed